src/react-components/inputs/input-with-proposed-values.jsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+import  React, { useState, useEffect } from 'react';
+import Input from '../../react-components/inputs/input';
+
+
+const InputWithProposedValues = ({ data, onClick, sortFunction, displayValue, ...props }) => {
```

Comment:
I'm thinking about this component name... maybe InputWithSuggestions is better? @sniedzielski what do you think? Do you have any other suggestions?

Reply:
'InputWithSuggestions' is a better one in my opinion.

Reply:
I agree with you, something was not right with this name for me, but I could not find a better proposition, I will change it

Reply:
Component name updated :)

Reply:
thx